commands/list_releases_test.go

```diff
@@ -43,6 +43,31 @@ func Test_ListReleases_Empty(t *testing.T) {
 	}
 }
 
+// Test_ListReleases_Connection_Unavailable simulates the situation where we
+// cannot reach the endpoint
+func Test_ListReleases_Connection_Unavailable(t *testing.T) {
+	dir, err := tempConfig("")
+	if err != nil {
+		t.Error(err)
+	}
+	defer os.RemoveAll(dir)
+
+	// needed to prevent search for the default cluster
+	args := listReleasesArguments{}
+	args.apiEndpoint = "http://localhost:45454"
+	args.token = "my-token"
```

Comment:
Why not ?
```golang
args := listReleasesArguments{
  apiEndpoint: "http://localhost:45454",
  token: "my-token",
}
```

Reply:
Good point, since I just changed this in another PR